package.json

```diff
@@ -118,5 +119,8 @@
     "@types/react-dom": {
       "optional": true
     }
+  },
+  "resolutions": {
+    "jackspeak": "2.1.1"
```

Comment:
Workaround for https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092